frontend/src/wells/views/WellDetail.vue

```diff
@@ -485,7 +485,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
                 striped
                 small
                 bordered
-                :items="well.aquifer_parameters_set"
+                :items="this.userRoles.wells.edit ? well.aquifer_parameters_set : this.well.aquifer_parameters_set.filter(param => !param.private)"
```

Comment:
This is a security concern. If users don't have the edit role, the private aquifers shouldn't be returned to the FE. Same for the aquifer map, if private aquifers can show up there.

Reply:
Corrections made, thank you!
![Screenshot 2024-11-19 at 1 54 23 PM](https://github.com/user-attachments/assets/7554c8c7-1c17-4e27-a012-03cc15e1a2c0)